cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java

```diff
@@ -45,38 +51,38 @@
  * that takes the IMap data and visualizes it.
  * <p>
  * Below is a diagram of the computation stages:
- *
- *                  ┌───────────────────┐
- *                  │Twitter Data Source│
- *                  └──────────┬────────┘
- *                             │
- *                             v
- *                 ┌──────────────────────┐
- *                 │FlatMap Relevant Coins│
- *                 └──────────┬───────────┘
- *                            │
- *                            v
- *               ┌─────────────────────────┐
- *               │Calculate Sentiment Score│
- *               └─────────────┬───────────┘
- *                             │
- *                             v
- *                   ┌──────────────────┐
- *                   │Group by Coin Name│
- *                   └────┬─────────┬───┘
- *                        │         │
- *                ┌───────┘         └─────────┐
- *                │                           │
- *                v                           v
- *    ┌────────────────────────┐   ┌────────────────────────┐
- *    │    Calculate 5min      │   │    Calculate 30sec     │
- *    │Average with Event Count│   │Average with Event Count│
- *    └───────────┬────────────┘   └─────────────┬──────────┘
- *                │                              │
- *                v                              v
- *  ┌───────────────────────────┐ ┌─────────────────────────────┐
- *  │  Write results to IMap    │ │  Write results to IMap      │
- *  └───────────────────────────┘ └─────────────────────────────┘
+ * <p>
```

Comment:
Please do not format the diagram, can you revert this ?

Reply:
Sorry for this. I couldn't notice that the formatting broke the diagram. I've fixed it.